ideascube/templates/ideascube/includes/tag_cloud.html

```diff
@@ -1,7 +1,7 @@
 {% load i18n ideascube_tags %}
 {% spaceless %}
 <div class="card tinted">
-    <h4><span class="theme learn">{% trans "browse" %}</span> {% trans "Topics" %}</h4>
+    <h4><span class="theme discover">{% trans "browse" %}</span> {% trans "Topics" %}</h4>
```

Comment:
Oh, I see. That's an eyesore indeed. 😆 

What tripped me up is that the CSS classes use the `discover` and `leanr` stuff from the home cards, which is really an abuse.

Anyway, changing all that would be too big a change, let's wait for the UI rework.

This specific change is approved, end of the thread as far as I'm concerned.